### PR TITLE
Add form_factor field to screenshot member

### LINKED ISSUE #55: consider ranaming "platform"
"platform" is also used in the [manifest spec](https://www.w3.org/TR/appmanifest/#platform-member) and has different semantics. 

"narrow" or "wide" also isn't exactly a platform.

How about renaming it to "target"? Or any other better suggestions here?

### PR DESCRIPTION
**Pull request by philloooo**
resolves #55 
This adds a `form_factor` field and move "wide" and "narrow" values from `platform` to `form_factor`

### CODE REVIEW

**Comment by philloooo** [comment]
@aarongustafson please take a look! I will submit the follow up CL for removing the `screenshot` field after this one is merged.

File: index.html [hunk 1]
```diff
@@ -467,6 +448,49 @@ <h3>
...
+      <p class="note" title="form_factor usage guidance for authors">
+        Authors should only use `form_factor` for instances where a screenshot
+        is not representative of a universal experience. For instance, if the
+        layout/design differs based on the screen orientation, setting "narrow" or
+        "wide" is advisable.
+      </p>
+      <p>
+        User agents might show as many (or as few) screenshots as they choose,
+        but shouldn't display screenshots that do not pertain to their
+        form_factor (e.g., Android phone shouldn't show wide form_factor
```
Comment (aarongustafson) [inline index.html:471]:
```suggestion
        form factor (e.g., Android phone shouldn't show "wide" `form_factor`
```

**Review by aarongustafson — CHANGES_REQUESTED** [review]

**Comment by aarongustafson** [comment]
Looks good @philloooo. Thanks!

**Comment by philloooo** [comment]
@aarongustafson  can you merge this?

File: index.html [hunk 2]
```diff
@@ -467,6 +448,49 @@ <h3>
         </p>
       </section>
     </section>
+    <section>
+      <h3>
+        `form_factor` member
+      </h3>
+      <p>
+        The <code><dfn class="export">form_factor</dfn></code> member is a
```
Comment (marcoscaceres) [inline index.html:456]:
```suggestion
        The <code><dfn data-for="manifest" class="export">form_factor</dfn></code> member is a
```

**Review by marcoscaceres — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -467,6 +448,49 @@ <h3>
...
+      <p class="note" title="form_factor usage guidance for authors">
+        Authors should only use `form_factor` for instances where a screenshot
+        is not representative of a universal experience. For instance, if the
+        layout/design differs based on the screen orientation, setting "narrow" or
+        "wide" is advisable.
+      </p>
+      <p>
+        User agents might show as many (or as few) screenshots as they choose,
+        but shouldn't display screenshots that do not pertain to their
+        form factor (e.g., Android phone shouldn't show "wide" `form_factor`
```
Comment (marcoscaceres) [inline index.html:471]:
```suggestion
        form factor (e.g., mobile phones shouldn't show "wide" `form_factor`
```

**Pull request merged**